<launch>
    <!--define two machines-->
    <!--<machine name="rpi" address="192.168.179.113" env-loader="/opt/ros/kinetic/env.sh" />-->
    <!--load parameters-->
    <rosparam file="$(find sechs_odrive)/cfg/axis_0.yaml" command="load"/>
    <rosparam file="$(find sechs_odrive)/cfg/axis_1.yaml" command="load"/>
    <rosparam file="$(find sechs_odrive)/cfg/axis_2.yaml" command="load"/>
    <rosparam file="$(find sechs_odrive)/cfg/axis_3.yaml" command="load"/>
    <rosparam file="$(find sechs_odrive)/cfg/axis_4.yaml" command="load"/>
    <!--load nodes-->
    <!--<node name="odrive_interface" pkg="sechs_odrive" type="odrive_interface.py" machine="rpi" />-->
    <!--delete parameters-->
    <!--<rosparam command="delete" param="axis_1" />-->
    <!--<node name="gui" pkg="sechs_gui" type="sechs_gui.py" />-->
</launch>